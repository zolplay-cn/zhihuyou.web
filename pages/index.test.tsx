import renderer from 'react-test-renderer'
import Home from './index'

describe('Home Page', () => {
  it('should match the snapshot', () => {
    const tree = renderer.create(<Home />).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
