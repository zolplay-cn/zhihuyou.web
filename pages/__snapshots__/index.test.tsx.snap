// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home Page should match the snapshot 1`] = `
<div>
  <main>
    <h1>
      Welcome to 
      <a
        href="https://nextjs.org"
      >
        Next.js!
      </a>
    </h1>
  </main>
  <footer>
    <a
      href="https://vercel.com?utm_source=create-next-app&utm_medium=default-template&utm_campaign=create-next-app"
      rel="noopener noreferrer"
      target="_blank"
    >
      Powered by
       
      <span>
        <div
          style={
            Object {
              "boxSizing": "border-box",
              "display": "inline-block",
              "margin": 0,
              "maxWidth": "100%",
              "overflow": "hidden",
              "position": "relative",
            }
          }
        >
          <div
            style={
              Object {
                "boxSizing": "border-box",
                "display": "block",
                "maxWidth": "100%",
              }
            }
          >
            <img
              alt=""
              aria-hidden={true}
              role="presentation"
              src="data:image/svg+xml;base64,PHN2ZyB3aWR0aD0iNzIiIGhlaWdodD0iMTYiIHhtbG5zPSJodHRwOi8vd3d3LnczLm9yZy8yMDAwL3N2ZyIgdmVyc2lvbj0iMS4xIi8+"
              style={
                Object {
                  "border": "none",
                  "display": "block",
                  "margin": 0,
                  "maxWidth": "100%",
                  "padding": 0,
                }
              }
            />
          </div>
          <noscript>
            <img
              alt="Vercel Logo"
              decoding="async"
              src="/vercel.svg"
              srcSet="/_next/image?url=%2Fvercel.svg&w=96&q=75 1x, /_next/image?url=%2Fvercel.svg&w=256&q=75 2x"
              style={
                Object {
                  "border": "none",
                  "bottom": 0,
                  "boxSizing": "border-box",
                  "display": "block",
                  "height": 0,
                  "left": 0,
                  "margin": "auto",
                  "maxHeight": "100%",
                  "maxWidth": "100%",
                  "minHeight": "100%",
                  "minWidth": "100%",
                  "objectFit": undefined,
                  "objectPosition": undefined,
                  "padding": 0,
                  "position": "absolute",
                  "right": 0,
                  "top": 0,
                  "width": 0,
                }
              }
            />
          </noscript>
          <img
            alt="Vercel Logo"
            decoding="async"
            src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7"
            style={
              Object {
                "border": "none",
                "bottom": 0,
                "boxSizing": "border-box",
                "display": "block",
                "height": 0,
                "left": 0,
                "margin": "auto",
                "maxHeight": "100%",
                "maxWidth": "100%",
                "minHeight": "100%",
                "minWidth": "100%",
                "objectFit": undefined,
                "objectPosition": undefined,
                "padding": 0,
                "position": "absolute",
                "right": 0,
                "top": 0,
                "width": 0,
              }
            }
          />
        </div>
      </span>
    </a>
  </footer>
</div>
`;
